Makefile

```diff
@@ -31,16 +31,16 @@ install: build
 	$(PYTHON) setup.py install -f
 
 test:
-	$(PYTHON) quipucords/manage.py test -v 2 quipucords
+	$(PYTHON) quipucords/manage.py test -v 2 quipucords/
 
 test-coverage:
-	coverage run --source=quipucords/ quipucords/manage.py test -v 2 quipucords;coverage report -m
+	coverage run --source=quipucords/ quipucords/manage.py test -v 2 quipucords/;coverage report -m
 
 lint-flake8:
 	flake8 . --ignore D203
 
 lint-pylint:
-	$(PYTHON) -m pylint --load-plugins=pylint_django quipucords/*/*.py
+	find . -name "*.py" | xargs $(PYTHON) -m pylint --load-plugins=pylint_django --disable=duplicate-code
```

Comment:
What code shows up when you enable the duplicate code checking? I would prefer to keep it enabled and spend a little more effort on refactoring if the warnings are at all reasonable.

Reply:
Only duplicate code is in the test files and I couldn't get the disable to work across files (several stackoverflows on this):

```
R:  1, 0: Similar lines in 2 files
==cli.auth.tests:21
==cli.tests:21
    def setUp(self):
        # Temporarily disable stderr for these tests, CLI errors clutter up
        # nosetests command.
        self.orig_stderr = sys.stderr
        sys.stderr = HushUpStderr()

    def tearDown(self):
        # Restore stderr
        sys.stderr = self.orig_stderr
 (duplicate-code)
R:  1, 0: Similar lines in 2 files
==cli.auth.tests:12
==cli.tests:12
import unittest
import sys
import six
from cli.cli import CLI
from cli.tests_utilities import HushUpStderr, redirect_stdout

 (duplicate-code)

```

Reply:
Okay, fine with me.